pages/docs/community/styleguide/links.md

```diff
@@ -0,0 +1,35 @@
+- Avoid using general words such as "click here" or "read more."
+- Make sure the link text accurately describes the target of the link. For example: **"Check out the latest version of AsyncAPI"** instead of **"click here for more information"**.
+- For external links, it is recommended to indicate that it is an external link by using the text **external link**.
+
+## External Links 🌐
+
+Linking to external websites is a common practice, but it can be tricky to do it in a way that's user-friendly. This section will show you how to indicate that a link is external and ensure that it opens in a new tab. By following these guidelines, you'll make it easier for your readers to navigate between your content and external resources without losing their place.
+
+- Use target=**"_blank"** attribute to open external links in a new tab. This makes it easier for the user to return to the original page.
+- Ensure that the external links open in a new tab or window, and not in the same tab, so as to avoid losing the original page. For example, if you click on an external link, a new tab should open and the original page should still be accessible.
+
+## Link Structure 📊
+
+Having a consistent and clear link structure can greatly improve the readability and flow of your content. In this section, you'll learn how to use a consistent link structure throughout your content and avoid using too many links in one section. By doing so, you can make your content more organized and easier to read, while also avoiding distractions or confusion.
+
+- Use a consistent link structure throughout the content. For example, using the same color and underline style for all links.
+- Make sure that the links are easily distinguishable from the rest of the text. You can use different colors for links so that links can be easily separated.
+- Avoid using too many links in one section, as it can be distracting and overwhelming for the reader. For example, having only 2-3 links in a section of text.
+
+## Conclusion 💡
+
+Yay!🎉 you maded till the end of the document, The conclusion section of this guide summarizes the key points and emphasizes the importance of following these guidelines. By making your links more effective and user-friendly, you'll enhance the overall user experience and improve the engagement and readability of your content. Whether you're writing for a blog ( [You can also publish a blog with AsyncAPI here](https://github.com/asyncapi/community/blob/master/contribute-blog-post.md) ), website, or other platform, these guidelines can help you create better links and better content.
+
+This Link styleguide will help you while creating the links, Also while designing the link structure we recommend you to focus on the link structure for a smooth user experience.
+
+Remember that the goal of linking is to help readers navigate your content and find the information they need, so always keep their needs in mind when designing your links.
```

Comment:
```suggestion
```

Let's delete the conclusion, that's unnecessary because this isn't a blog post. Let's keep all info explaining the purpose of this section in the Introduction section. :)

Reply:
Okay, done